### PR TITLE
Remove node 6 and callback support

### PR DESCRIPTION
**Pull request by fcastilloec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,20 +172,12 @@ installer(options)
...
-    process.exit(1)
-  }
-
-  console.log(`Successfully created package at ${options.dest}`)
-})
-```
 
 You'll end up with the package at `dist/installers/app-0.0.1-setup.exe`.
 
+_Note: As of 2.0.0, the Node-style callback pattern is no longer available. You can use [`nodeify`](https://npm.im/nodeify) if this is required for your use case._
```
Comment (malept) [inline README.md]: Link to `util.callbackify` in the Node.js docs instead

File: package.json [hunk 2]
```diff
@@ -26,32 +26,31 @@
...
   },
   "dependencies": {
     "debug": "^4.1.0",
-    "electron-installer-common": "^0.6.1",
-    "fs-extra": "^7.0.1",
-    "glob": "^7.1.3",
+    "electron-installer-common": "^0.6.3",
+    "fs-extra": "^8.0.1",
+    "glob": "^7.1.4",
     "glob-promise": "^3.4.0",
```
Comment (malept) [inline package.json]: I've found that `glob-promise` isn't really needed, you just need to `promisify(glob)`.

**Comment by fcastilloec** [comment]
I guess we can make another PR for `async/await` just to keep things a little neater.

If I have time, I can help with removing node 6 from the `-common` module. If that takes too long, we can update that dependency for this module on the `async/await` PR.

**Comment by malept** [comment]
It's mostly done: https://github.com/electron-userland/electron-installer-common/tree/drop-node-6 _(Forgot to fix the docs, apparently.)_

**Pull request closed without merging**